sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/DocumentHelper.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Models;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals
+{
+    /// <summary>
+    /// Helper class used to convert Activity to the LiveMetrics document schema.
+    /// </summary>
+    internal static class DocumentHelper
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static RemoteDependency ConvertToRemoteDependency(Activity activity)
+        {
+            string urlFull = string.Empty, httpResponseStatusCode = string.Empty;
+
+            foreach (ref readonly var tag in activity.EnumerateTagObjects())
+            {
+                if (tag.Value == null)
```

Comment:
This check is not needed, as per the spec values are mandatory in the below fields.

Reply:
But the Tags are KVP<string, object?>. We have to do the null check somewhere and helps skip any empty KVpairs.

Reply:
This was removed in favor of `ActivityTagsProcessor`. Discussed offline and we may replace this with something more optimized at a later date.